lib/pages/app_grid/app_grid_dashboard_item.dart

```diff
@@ -28,37 +33,70 @@ class AppGridDashboardItem extends StatelessWidget {
+          onEnter: (_) =>
+              hoverColor.value = AppGridDashboardViewStyle.hoverColor,
+          onExit: (_) => hoverColor.value = Colors.transparent,
+          child: ValueListenableBuilder<Color>(
+            valueListenable: hoverColor,
+            builder: (context, color, child) {
+              return Container(
+                width: AppGridDashboardItemStyle.itemWidth,
+                padding: AppGridDashboardItemStyle.padding,
+                decoration: BoxDecoration(
+                  color: color,
+                  borderRadius: AppGridDashboardItemStyle.borderRadius,
+                ),
+                child: Column(
+                  children: [
+                    if (app.publicIconUri != null)
+                      Image.network(
+                        app.publicIconUri.toString(),
                         width: AppGridDashboardItemStyle.iconAppSize,
                         height: AppGridDashboardItemStyle.iconAppSize,
                         fit: BoxFit.fill,
+                        errorBuilder: (_, error, stackTrace) {
+                          return Container(
+                            width: AppGridDashboardItemStyle.iconAppSize,
+                            height: AppGridDashboardItemStyle.iconAppSize,
+                            color: Theme.of(context).hintColor,
+                          );
+                        },
                       )
-                    : Image.asset(
-                        ImagePaths.getConfigurationImagePath(app.iconName),
-                        width: AppGridDashboardItemStyle.iconAppSize,
-                        height: AppGridDashboardItemStyle.iconAppSize,
-                        fit: BoxFit.fill,
+                    else
+                      app.iconName.endsWith("svg")
+                          ? SvgPicture.asset(
+                              ImagePaths.getConfigurationImagePath(
+                                app.iconName,
+                              ),
+                              width: AppGridDashboardItemStyle.iconAppSize,
+                              height: AppGridDashboardItemStyle.iconAppSize,
+                              fit: BoxFit.fill,
+                            )
+                          : Image.asset(
+                              ImagePaths.getConfigurationImagePath(
+                                app.iconName,
+                              ),
+                              width: AppGridDashboardItemStyle.iconAppSize,
+                              height: AppGridDashboardItemStyle.iconAppSize,
+                              fit: BoxFit.fill,
+                            ),
+                    Padding(
+                      padding: AppGridDashboardItemStyle.textPadding,
+                      child: Text(
+                        app.appName.contains('Twake')
+                            ? app.appName.split('Twake ').last
+                            : app.appName,
```

Comment:
IMO: Create an extension for handle this case.

Reply:
Done